src/Makefile.am

```diff
@@ -60,6 +60,7 @@ decode-teredo.c decode-teredo.h \
 decode-udp.c decode-udp.h \
 decode-vlan.c decode-vlan.h \
 decode-mpls.c decode-mpls.h \
+decode-template.c decode-template.h \
```

Comment:
A bit torn here. I would like this to be part of the regular build testing. Planning to create a QA test that also tests the scripts themselves, but that would exclude it from static code analysis unless we do a lot of trickery first. So think I'll keep it for now and we can revisit later.